packages/solana/src/connectors/walletConnectConnector.ts

```diff
@@ -7,7 +7,7 @@ import { UniversalProviderFactory } from './universalProvider.js'
 import { BaseConnector } from './baseConnector.js'
 
 import type { Signer } from '@solana/web3.js'
-import type UniversalProvider from '@walletconnect/universal-provider'
```

Comment:
why this?

Reply:
This actually breaks in some bundlers (default import of UP), it should be a name import.

Reply:
@glitch-txs can we make a lint or tsconfig role to disallow this?